google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -951,7 +951,14 @@ def download path = nil, verify: :md5, encryption_key: nil, range: nil,
         end
 
         ##
-        # Copy the file to a new location.
+        # Copies the file to a new location. Metadata from the source object will
+        # be copied to the destination object unless a block is provided.
```

Comment:
Metadata excluding ACL from the source object will be copied to...

Reply:
Will do.